File: Java/UberVan.java
package Java;

import java.util.ArrayList;
import java.util.Map;

public class UberVan extends Car {
    Map<String, Map<String, Integer>> typeCarAccepted;
    ArrayList<String> seatsMaterial;
    private Integer passengers;

    public UberVan(String license, Account driver) {
        super(license, driver);
    }

    @Override
    public void setPassenger(Integer passenger) {
        if (passenger == 6) {
            this.passengers = passenger;
        } else {
            System.out.println("Numero de pasajeros no validos");
        }
    }
}

File: PHP/UberVan.php
<?php
require_once('Car.php');

class UberVan extends Car{
    public $typeCarAccepted;
    public $seatsMaterial;
    protected $passengers;

    public function __construct($license, $driver, $typeCarAccepted, $seatsMaterial){
        parent::__construct($license, $driver);
        $this->typeCarAccepted = $typeCarAccepted;
        $this->seatsMaterial = $seatsMaterial;
    }

    public function setPassenger($passengers) {
    
    if ($passengers == 6) {
        $this->passenger = $passengers;
    }
    else {
        echo "Numero de pasajeros no valido";
    }

}
}